io/mockk/proxy/MockKAgentFactory java.lang.NoClassDefFoundError: io/mockk/proxy/MockKAgentFactory

### Prerequisites

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Expected Behavior
Tests using a combination of Spek and mockk should be able to run on AndroidStudio.
I'd like to know if there are any solutions.

### Current Behavior
The test cannot be executed due to an error.
But, the test with the command(`./gradlew test`) succeeds

I've tried reinstalling plugins, deleting AndroidStudio cache, and rebooting.

### Failure Information (for bugs)
It happens when I test using the Spek Framework plugin on the project AndroidStudio in the link below.
https://github.com/mitohato/mockk-test

#### Steps to Reproduce

Run the code shown below as a Spek Framework plugin test on AndroidStudio.
[Spek Framework plugin](https://plugins.jetbrains.com/plugin/10915-spek-framework)

#### Context

* MockK version: 1.11.0
* Spek version: 2.0.15
* OS: macOS Catalina 10.15.6
* Kotlin version: 1.4.32
* JDK version: java 15.0.1 2020-10-20
* JUnit version: 1.1.2
* Type of test: unit test
* Spek Framework plugin version: 2.0.16-IJ2020.1

It happened with MockK 1.10.0 or later versions, and was fine with MockK 1.9.3.


#### Failure Logs
<img width="2048" alt="screenshot" src="https://user-images.githubusercontent.com/19385268/115840610-d43f6800-a456-11eb-8fc6-18120882e6b5.png">

#### Stack trace

```
io/mockk/proxy/MockKAgentFactory
java.lang.NoClassDefFoundError: io/mockk/proxy/MockKAgentFactory
	at io.mockk.impl.JvmMockKGateway.<init>(JvmMockKGateway.kt:185)
	at io.mockk.impl.JvmMockKGateway.<clinit>(JvmMockKGateway.kt:173)
	at io.mockk.MockKKt.every(MockK.kt:712)
	at com.example.spektest.ExampleUnitTest$1$1$1$1.invoke(ExampleUnitTest.kt:26)
	at com.example.spektest.ExampleUnitTest$1$1$1$1.invoke(ExampleUnitTest.kt:16)
	at org.spekframework.spek2.runtime.scope.TestScopeImpl.execute(scopes.kt:136)
	at org.spekframework.spek2.runtime.Executor$execute$result$2$exception$1$job$1.invokeSuspend(Executor.kt:84)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:271)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:79)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:54)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:36)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
	at org.spekframework.spek2.runtime.TaskRunner$runTask$1.run(executors.kt:26)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: io.mockk.proxy.MockKAgentFactory
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:419)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:352)
	... 21 more


```

#### Minimal reproducible code (the gist of this issue)

```kotlin
// -----------------------[ GRADLE DEFINITIONS ] -----------------------
dependencies {
    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.32")
    implementation("org.jetbrains.kotlin:kotlin-reflect:1.4.32")

    androidTestImplementation("androidx.test.ext:junit:1.1.2")
    testImplementation("io.mockk:mockk:1.11.0")
    testImplementation("org.spekframework.spek2:spek-dsl-jvm:2.0.15")
    testImplementation("org.spekframework.spek2:spek-runner-junit5:2.0.15")
    testImplementation("com.google.truth:truth:1.1.2")
}
// -----------------------[ YOUR CODE STARTS HERE ] -----------------------

package com.example.spektest

import com.google.common.truth.Truth
import io.mockk.every
import io.mockk.mockk
import io.mockk.verify

import org.spekframework.spek2.Spek
import org.spekframework.spek2.style.gherkin.Feature

class ExampleUnitTest : Spek({
    Feature("Example") {
        val interfaceForTarget by memoized { mockk<InterfaceForTarget>() }
        val target by memoized { Target(interfaceForTarget) }
        Scenario("test") {
            var result: Int? = null
            val expect = 10
            Given("return 5") {
                every {
                    interfaceForTarget.getValue()
                } returns 5
            }
            When("execute()") {
                result = target.execute()
            }
            Then("same value") {
                Truth.assertThat(result).isEqualTo(expect)
            }
            Then("call getValue") {
                verify {
                    interfaceForTarget.getValue()
                }
            }
        }
    }
})

interface InterfaceForTarget {
    fun getValue(): Int
}

class Target(
    private val interfaceForTarget: InterfaceForTarget
) {
    fun execute(): Int = interfaceForTarget.getValue() * 2
}

// -----------------------[ YOUR CODE ENDS HERE ] -----------------------
```

[Full code is here](https://github.com/mitohato/mockk-test)

--- a/README.md
+++ b/README.md
@@ -96,13 +96,19 @@ All you need to get started is just to add a dependency to `MockK` library.
 <tr>
 <td><a href="ANDROID.md"><img align="top" src="doc/robot-small.png" height="20" alt="android"/> Unit</a></td>
 <td>
-    <pre>testImplementation "io.mockk:mockk:{version}"</pre>
+<pre>
+testImplementation "io.mockk:mockk:{version}"
+testImplementation "io.mockk:mockk-agent-jvm:{version}"
+</pre>
 </td>
 </tr>
 <tr>
 <td><a href="ANDROID.md"><img align="top" src="doc/robot-small.png" height="20" alt="android"/> Instrumented</a></td>
 <td>
-    <pre>androidTestImplementation "io.mockk:mockk-android:{version}"</pre>
+<pre>
+androidTestImplementation "io.mockk:mockk-android:{version}"
+androidTestImplementation "io.mockk:mockk-agent-jvm:{version}"
+</pre>
 </td>
 </tr>
 <tr>

